flink-end-to-end-tests/flink-end-to-end-tests-sql/pom.xml

```diff
@@ -212,6 +212,16 @@
                     <scope>test</scope>
                 </dependency>
             </dependencies>
+            <dependencyManagement>
+                <dependencies>
+                    <dependency>
+                        <!-- dependency convergence -->
+                        <groupId>com.nimbusds</groupId>
```

Comment:
we need to add `com.squareup.okio:okio:1.17.2` too
https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=40167&view=logs&j=b1fcf054-9138-5463-c73c-a49979b9ac2a&t=9291ac46-dd95-5135-b799-3839e65a8691&l=24139